code/game/objects/items/storage/belt.dm

```diff
@@ -82,6 +82,26 @@
 	new /obj/item/tool/wirecutters(src)
 	new /obj/item/t_scanner(src)
 
+/obj/item/storage/belt/medical_small
+	name = "\improper M276 pattern light medical rig"
+	desc = "The M276 is the standard load-bearing equipment of the TGMC. It consists of a modular belt with various clips. This lightweight configuration is often used for bulk carrying of medical supplies by marines."
+	icon_state = "medicalbelt_small"
+	item_state = "medicbag"
+	storage_slots = 15
+	max_storage_space = 30
+	max_w_class = 3
+	can_hold = list(
+		/obj/item/healthanalyzer,
+		/obj/item/reagent_containers/glass/bottle,
+		/obj/item/reagent_containers/pill,
+		/obj/item/reagent_containers/syringe,
+		/obj/item/storage/syringe_case,
+		/obj/item/storage/pill_bottle,
+		/obj/item/stack/medical,
+		/obj/item/reagent_containers/hypospray,
+		/obj/item/reagent_containers/dropper,
```

Comment:
I'm not wild about this in the first place, but if you're designing expanded medical storage for standard marines to use it needs to be limited to things standard marines should be using: packets, autoinjectors, stacks, and not a whole lot else. Total space is on the high end, but in keeping with pouches.